shared/gh/js/views/tenant-admin/gh.datepicker.js

```diff
@@ -470,6 +483,29 @@ define(['gh.core', 'moment', 'moment-timezone', 'clickover', 'jquery-datepicker'
     };
 
 
+    /////////////////
+    //  UTILITIES  //
+    /////////////////
+
+    /**
+     * Return the number of weeks for a date, if it occurs in a term
+     *
+     * @param  {String}    startDate    The start date of the term to return the number of weeks for
+     * @return {Number}                 The number of weeks
+     * @private
+     */
+    var getNumberOfWeeks = function(startDate) {
```

Comment:
Not really, since this function checks if the chosen date is within a term. If so, then it returns the number of weeks. Otherwise we would have to do the term check before executing this function on several places in the code..